.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs

```diff
@@ -188,6 +216,7 @@ private ParseResult MergeCompoundUnit(ExtractResult compoundResult)
 
                         count = 0;
                         idx -= 1;
+                        numberValue = null;
```

Comment:
Why?

Reply:
> Why?

Set this to the default value in case of next unit without value get previous unit value. Such as "50 dollar RMB", the RMB will get the 50 as value.